### PR TITLE
IDE-3500 add project property page for "gradle watch"

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by jtydhr88** [comment]
jenkins retest

**Comment by yanan** [comment]
jenkins retest

File: tools/plugins/com.liferay.ide.core/plugin.xml [hunk 1]
```diff
@@ -91,6 +91,17 @@
 			</run>
 		</runtime>
 	</extension>
+	<extension
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.core/plugin.xml]: I don't want to add another nature.  We should instead use project settings.

Comment (gamerson) [reply tools/plugins/com.liferay.ide.core/plugin.xml]: Lets discuss this next week.

**Pull request closed without merging**